__tests__/index.tsx

```diff
@@ -1,7 +1,15 @@
+/**
+ * @jest-environment node
+ */
+
 import * as React from 'react';
-import { shallow, render, mount } from 'enzyme';
+import { shallow, configure } from 'enzyme';
 import Circle, {CircleProps} from '../src';
 
+const Adapter = require("enzyme-adapter-react-16");
+
+configure({ adapter: new Adapter() });
```

Comment:
I couldn't get it running without these changes due to these bugs/errors [here](https://stackoverflow.com/questions/51554366/jest-securityerror-localstorage-is-not-available-for-opaque-origins) and [here](https://github.com/airbnb/enzyme/issues/1284#issuecomment-389522919)